content/pokt/wallets.md

```diff
@@ -82,7 +83,7 @@ For more convenience, you may consider using a password manager that can store e
 
 ### Import wallet
 
-The [official wallet web app](https://wallet.pokt.network) has two methods of importing your wallet. The Key File is the more secure method of importing your wallet since it is encrypted by your passphrase.
+The [official wallet web app](https://wallet.pokt.network) has three methods of importing your wallet. The Key File is the more secure method of importing your wallet since it is encrypted by your passphrase. 
```

Comment:
It was not clear the correct guidance to use in place of:
> The Key File is the more secure method of importing your wallet since it is encrypted by your passphrase.